src/SIM/SIM2D.C

```diff
@@ -82,7 +82,7 @@ bool SIM2D::addConnection (int master, int slave, int mIdx,
   int lslave = this->getLocalPatchIndex(slave);
   if (lmaster > 0 && lslave > 0)
   {
-    if (dim < 1) return false;
+    if (dim == 0) return true; // ignored in serial
```

Comment:
Does negative dim have some interpretation for 2D? If not, maybe change to `if (dim < 1);` to make it similar as the 3D case

Reply:
changed.